test/datetime.cc

```diff
@@ -645,23 +645,13 @@ TEST(DateTime, TimezoneAliases) {
   const auto& dt_db = DateTime::get_tz_db();
   // map of alias and target names
   // this can be old deprecated timezone name (pre-2023) or renamed timezones (after 2023)
-  std::vector<std::pair<std::string, std::string>> pairs =
-      {{"Etc/Zulu", "Etc/UTC"},
-       {"Etc/GMT-0", "Etc/GMT"},
-       {"ROC", "Asia/Taipei"},
-       {"GB", "Europe/London"},
-       {"NZ-CHAT", "Pacific/Chatham"},
-       {"Asia/Ujung_Pandang", "Asia/Makassar"},
-       {"Africa/Bamako", "Africa/Abidjan"},
-       {"Africa/Kampala", "Africa/Nairobi"},
-       {"Asia/Kuala_Lumpur", "Asia/Singapore"}, // ^ deprecated 2018 tz
```

Comment:
I removed these, as they're not making sense to test. they're from pre-tz-upgrade times and also their deprecated names have their own indices in the hard-coded tz index map.